live-examples/css-examples/pseudo-element/after.html

```diff
@@ -1,6 +1,6 @@
 <p>
   The sailfish is named for its sail-like dorsal fin and is widely considered the fastest fish in the ocean.
-  <a href="https://en.wikipedia.org/wiki/Sailfish">You can read more about it here</a>.
+  <a href="https://example.com/wiki/Sailfish">You can read more about it here</a>.
```

Comment:
people might actually want to learn about sailfish. We should keep the original here.

Reply:
@estelle @bsmth That's not the purpose of this example and MDN.